nntrainer/graph/network_graph.cpp

```diff
@@ -227,6 +228,12 @@ int NetworkGraph::realizeActivationType(Layer &current) {
 
   ActivationType act = current.getActivationType();
 
+  if (current.getType() == RNNLayer::type) {
+    // No need to add activation layer for RNN Layer
+    current.setActivation(act);
```

Comment:
`none` activation layer is also supported?

Reply:
I'm thinking of. but later PR, default activation is tanh.

Reply:
I think `none` must not be supported. Can be opened later if requested.